charmcraft/templates/init-kubernetes/tests/unit/test_charm.py.j2

```diff
@@ -10,7 +10,16 @@ from charm import SERVICE_NAME, {{ class_name }}
 
 CHECK_NAME = "service-ready"  # Name of Pebble check in the mock workload container.
 
-base_layer: pebble.LayerDict = {"services": {SERVICE_NAME: {}}, "checks": {CHECK_NAME: {}}}
+base_layer: pebble.LayerDict = {
+    "services": {SERVICE_NAME: {}},
+    "checks": {
+        CHECK_NAME: {
+            "startup": "enabled",
```

Comment:
Similarly here, `override` is required so should be included I think. There ought to be one of the checks (http/tcp/exec) as well, although that gets to the same issue as `command` with services, that you have to define something and there's probably not an obvious choice for a template.